packages/inter-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -973,7 +974,9 @@ test('interest on multiple vaults', async t => {
   // Advance 8 days, past one charging and recording period
   await manualTimer.tickN(8);
 
-  const assetUpdate = await E(assetNotifier).getUpdateSince();
+  const assetUpdate = (
+    await E(E(aethCollateralManager).getSubscriber()).subscribeAfter()
```

Comment:
What does this change do?

Reply:
This open codes getting the latest value from what `getSubscriber()` returns from the CollateralManager. Not a functional change. Just needed to 1) get the stream from the collateral manager instead of the offer result, and 2) read a Subscriber instead of a Notifier.